data/schema/schema_v1.json

```diff
@@ -533,16 +534,19 @@
                               "type": "integer"
                             },
                             "alias": {
-                              "type": "string"
+                              "type": "string",
+                              "description": "Allows another name for a variable to be used."
```

Comment:
Doesn't quite capture what this is for, something like "An alias that can be used when piping this answer" would be better.

Reply:
Done